.github/workflows/tests.yml

```diff
@@ -0,0 +1,86 @@
+name: GitHub Actions CI
```

Comment:
I don't know if we need this file - I cannot see anything preventing random users DOS-ing by starting many actions.

Reply:
This came from the upstream 
I am encountering issues running the tests as then are targetting macOS 11 which as per the below link is not open for new registrations
 [https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners]